pkg/kubectl/cmd/cmd.go

```diff
@@ -225,25 +227,25 @@ func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 		{
 			Message: "Basic Commands (Beginner):",
 			Commands: []*cobra.Command{
-				NewCmdCreate(f, out, err),
+				NewCmdCreate(f, out, errOut),
```

Comment:
We should be passing `out` and `errOut` to every single command, as a rule of thumb. Not all our commands write errors to stderr, but should.